app/src/main/java/com/codesoom/assignment/utils/JwtUtil.java

```diff
@@ -0,0 +1,70 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.security.Key;
+import java.util.Date;
+
+/**
+ * Jwt 토큰을 관리한다.
+ */
+public class JwtUtil {
+    private final Key key;
+    private final long validTime;
+
+    public JwtUtil(@Value("${jwt.secret}") String secret,
+                   @Value("${jwt.valid-time}") long validTime) {
+        this.key = Keys.hmacShaKeyFor(secret.getBytes());
+        this.validTime = validTime;
+    }
+
+    /**
+     * 주어진 사용자 id로 Jwt 토큰을 생성한다.
+     *
+     * @param userId - 사용자 식별자
+     * @return - 생성된 토큰
+     */
+    public String encode(Long userId) {
+        Date now = new Date();
+        Date expiration = calculationExpiration(now, validTime);
+
+        return Jwts.builder()
+                .claim("userId", userId)
+                .setIssuedAt(now)
+                .setExpiration(expiration)
+                .signWith(key)
+                .compact();
+    }
+
+    private Date calculationExpiration(Date now, long validTime) {
+        return new Date(now.getTime() + validTime);
+    }
+
+    /**
+     * 주어진 토큰을 복호화한 정보를 리턴한다.
+     *
+     * @param token - 토큰
+     * @return 복호화한 정보
+     * @throws InvalidTokenException 토큰이 유효하지 않을 경우
+     */
+    public Claims decode(String token) throws InvalidTokenException {
+        if (token == null || token.isBlank()) {
```

Comment:
```suggestion
        if (Strings.isBlank(token) {
```

nit : 항상 하는 널체크가 귀찮다면 이렇게 하는 것도 좋겠습니다! 
대표적인 문자열 관련 유틸모임은 apache commons lang 을 전통적으로 많이 사용합니다.
위 예시는 log4j에 구현체가 있길래 보여드린 예시입니다.

Reply:
nit가 nitpicking(중요하지 않지만 작은 변화)을 의미하는군요!  

[log4j](https://ko.wikipedia.org/wiki/Log4j) (아파치 로깅 유틸리티)

[Apache Commons](https://commons.apache.org/proper/commons-lang/) 라는 것도 있다니.. 아파치가 오픈소스 재단(?)인 줄 알았는데 개발 표준처럼 참고 용도로도 쓰이는 모양이네요~?

오우 하나하나 써 주시는 단어가 신세계입니다~가르침 감사합니다!